modules/acquia_cms_article/config/optional/views.view.article_cards.yml

```diff
@@ -52,7 +53,10 @@ display:
           items_per_page: 7
           offset: 0
       style:
-        type: default
+        type: cohesion_layout
+        options:
+          views_template: view_tpl_article_cards_slider
+          master_template: __none__
```

Comment:
@panshulK - We can't update views template directly here, this will throw error when site install with standard profile or where site studio is not preset, instead we can update this configuration based on modules availability like we have done for other configuration using EventSubscriber. 
You can take reference from here: https://github.com/acquia/acquia_cms/blob/develop/modules/acquia_cms_common/src/EventSubscriber/ConfigEventsSubscriber.php#L54-L70
cc: @vishalkhode1

Reply:
Thanks for sharing this @chandan-singh7929 , checking!

Reply:
Hey @chandan-singh7929 
Reverted the changes. Please review!

Thanks